tests/phpunit/tests/http/includeOldRequestsClass.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/**
+ * Tests that the old Requests class is included
+ * for plugins or themes that still use it.
+ *
+ * @group http
+ */
+class Tests_HTTP_IncludeOldRequestsClass extends WP_UnitTestCase {
+
+	/**
+	 * Tests that the old Requests class is included for plugins or themes that still use it.
+	 *
```

Comment:
```suggestion
	/**
```
Test method description isn't needed as it's redundant. Removing to simplify and for consistency.